tests/Dockerfile.almalinux-ArcticSphynx

```diff
@@ -0,0 +1,200 @@
+#
+# Sysbox Test Container Dockerfile (CentOS-8 image)
```

Comment:
Please update this with the proper distro name/release.

Also, I'm not too familiarized with AlmaLinux but it looks like their numbered based release-tagging convention (e.g. 8.5) may be more intuitive/known that the code based one (e.g. Arctic Sphynx). If that's the case, I would suggest to rename this file to `Dockerfile.almalinux-8.5`.

Reply:
The distro name has been correctly updated.

Regarding the release tagging on almalinux. When typing `make sysbox` the Makefile is searching for a file in tests directory with the name
Dockerfile.$(IMAGE_BASE_DISTRO)-$(IMAGE_BASE_RELEASE)

IMAGE_BASE_DISTRO comes from line [92](https://github.com/nestybox/sysbox/blob/master/Makefile#L92) in Makefile
IMAGE_BASE_RELEASE comes from line [103](https://github.com/nestybox/sysbox/blob/master/Makefile#L103) in Makefile. So the ArcticSphynx naming is because of the IMAGE_BASE_RELEASE which propably it could be added in the makefile like centos, fedora and redhat to get the naming correctly. What do you think ?

Reply:
Yes, if the 'correct' release name (e.g. 8.5) is printed when we add "almalinux" to the list of distros in line [100](), then please do that.

Reply:
Makefile was updated and the file name is now called `Dockerfile.almalinux-8` which matches the `Dockerfile.centos-8` file